etc/profile-m-z/npm.profile

```diff
@@ -0,0 +1,60 @@
+# Firejail profile for npm
+# Description: The Node.js Package Manager
```

Comment:
Might be good to add quiet here to avoid making npm's already pretty verbose output even noisier.